webcompat/templates/issue.html

```diff
@@ -2,34 +2,61 @@
 {% block title %}Issue #{{ number }}{% endblock %}
 {% block body %}
   {% include "nav/nav.html" %}
-  <main class="wc-UISection wc-UISection--backgroundColor" role="main"
+  <main class="grid grid-show-gap" role="main"
         data-issue-number={{ number }}
         data-repo-path="{{ config['ISSUES_REPO_URI'] }}"
         data-statuses="{{ config['JSON_STATUSES'] }}"
         >
-    <div class="wc-Issue">
-      <section class="wc-UIContent js-Issue is-hidden">
-        {% include "issue/issue-aside.html" %}
-        {% include "issue/issue-information.html" %}
-        <div class="wc-Issue-commentList js-Issue-commentList">
+
+<!-- <section " class="grid">
+  <div class="grid-row issue label-closed">
+    <article class="grid-cell x2 issue-description">
+      <p class="issue-number"> Issue 71234: </p>
+      <p class="issue-url"> https://unicornjs.com </p>
+      <p class="issue-type"> Audio / media does not play </p>
+      <p class="issue-date-open"> Opened: 2017-11-12 </p>
+      <p class="issue-date-closed"> Closed: 2017-11-14 </p>
+      <p class="issue-date-comments"> Comments: 12 </p>
+    </article>
+    <aside class="grid-cell x1 issue-labels" >
+      <p>
+        <span class="label label-browser"> browser-firefox </span>
+        <span class="label label-device"> device-mobile </span>
+        <span class="label label-os"> od-android </span>
```

Comment:
(should this just be removed? It looks like placeholder content)

Reply:
The `issue-description` section is something, I might want to keep in as this was the goal to achieve, but tricky as we discussed over IRC because we would need to change the markup there. I have the feeling this might be forgotten later on. 
(Especially as we will scrap the design documentation and auto-generate one from the markup we have right now).

But the `issue-labels`  section needs to go. :)